plasmapy/physics/tests/test_parameters.py

```diff
@@ -831,23 +825,23 @@ def test_upper_hybrid_frequency():
     omega_uh = upper_hybrid_frequency(B, n_e=n_e)
     omega_ce = gyrofrequency(B)
     omega_pe = plasma_frequency(n=n_e)
-    assert omega_ce.unit == u.rad / u.s
-    assert omega_pe.unit == u.rad / u.s
-    assert omega_uh.unit == u.rad / u.s
-    LHS = omega_uh**2
-    RHS = omega_ce**2 + omega_pe**2
+    assert omega_ce.unit.is_equivalent(u.rad / u.s)
+    assert omega_pe.unit.is_equivalent(u.rad / u.s)
+    assert omega_uh.unit.is_equivalent(u.rad / u.s)
+    LHS = omega_uh ** 2
```

Comment:
I think all caps might not be a good variable name. It implicitly distinguishes it from other variables and gets highlighted differently in Github's syntax highlighting.